package/src/specs/RNCandle.nitro.ts

```diff
@@ -344,6 +344,59 @@ export type ToolCall = {
   arguments: string;
 };
 
+export type LinkedAccountRef = {
+  linkedAccountID: string;
+};
+
+export type AssetAccountRef = {
+  linkedAccountID: string;
+  assetKind: string; // "fiat" | "stock" | "crypto"
```

Comment:
This should be generate-able, it's just a string enum

Reply:
Causes a compiler error with the nitro codegen